pages/forms/TripForm/[id].js

```diff
@@ -2,12 +2,16 @@ import CancelButton from "@/components/CancelButton";
 import CreateButton from "@/components/CreateButton";
 import { useRouter } from "next/router";
 import useSWR from "swr";
+import { useState } from "react";
 
 const fetcher = (...args) => fetch(...args).then((res) => res.json());
```

Comment:
The fetcher should be in `<SWRConfig>` inside `_app.js`

Reply:
We will keep that in mind for our refactoring!